coordinator/src/routes.rs

```diff
@@ -92,8 +93,8 @@ pub fn router(
         .route("/api/trade", post(post_trade))
         .route("/api/register", post(post_register))
         .route("/api/admin/balance", get(get_balance))
-        .route("/api/admin/channels", get(list_channels))
-        .route("/api/channels", post(open_channel))
+        .route("/api/admin/channels", get(list_channels).post(open_channel))
+        .route("/api/channels", post(channel_faucet))
```

Comment:
I think we can remove that endpoint. After merging https://github.com/get10101/10101/pull/816 this shouldn't be used anymore.